### PR TITLE
[#OSF-7930]Sorting OSF Meetings instances in Admin app

### PR DESCRIPTION
**Pull request by chennan47**
## Purpose
Sorting OSF Meetings instances in Admin app

## Changes
Now all the meetings in admin app should appear in alphabet order

## Ticket
https://openscience.atlassian.net/browse/OSF-7930

### CODE REVIEW

File: admin/meetings/views.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ class MeetingListView(PermissionRequiredMixin, ListView):
     raise_exception = True
 
     def get_queryset(self):
-        return Conference.find()
+        return sorted(Conference.find(), key=lambda meeting: meeting['name'])
```
Comment (brianjgeiger) [inline admin/meetings/views.py]: Don't use python for this; you can do the ordering on the query itself.

File: admin/meetings/views.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ class MeetingListView(PermissionRequiredMixin, ListView):
     raise_exception = True
 
     def get_queryset(self):
-        return Conference.find()
+        return Conference.find().order_by('name')
```
Comment (alexschiller) [inline admin/meetings/views.py]: Driveby: `.find` is a modm style wrapper for django functions: https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/base.py#L98

Just use `Conference.objects.all().order_by('name')`

Comment (chennan47) [inline admin/meetings/views.py]: Done in d9db1be
